Sources/DockProgress/util.swift

```diff
@@ -106,3 +110,29 @@ extension NSBezierPath {
 		self.init(roundedRect: rect, xRadius: cornerRadius, yRadius: cornerRadius)
 	}
 }
+
+final class VerticallyCenteredTextLayer: CATextLayer {
+	convenience init(frame rect: CGRect, center: CGPoint) {
+		self.init()
+		frame = rect
+		frame.center = center
+		alignmentMode = .center
+		truncationMode = .end
```

Comment:
This is also unrelated to vertically centering.

Reply:
Correct, I just thought it was cleaner to put it inside the subclass, I could either move it somewhere else (where it probably doesn't add much value) or rename the subclass?

Reply:
The subclass is useful for other situations too though. I wouldn't rename it. Just set the props where it's used.